packages/vite-plugin-cloudflare/src/runner/module-runner.ts

```diff
@@ -69,7 +77,10 @@ export async function createModuleRunner(
 				Object.freeze(context.__vite_ssr_exports__);
 			},
 			async runExternalModule(filepath) {
-				filepath = filepath.replace(/^file:\/\//, '');
+				return new Error(
+					`[Error] Trying to import external module (only internal modules are allowed): ${module.id}`,
+				);
```

Comment:
This does not work 😕 

I think the Vite module runner is swallowing the error at some point before it's returned to the user

I'm thinking of parking this effort for now and come back when we do have inspectors working, as that would really really simplify the investigation needed here